### PR TITLE
feat:Attribute Panel Block Selection Style

### PR DESCRIPTION
**Pull request by ianxinnew**
English | [简体中文](https://github.com/opentiny/tiny-engine/blob/develop/.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.zh-CN.md)

# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-engine/blob/develop/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Built its own designer, fully self-validated

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

### What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
	- Introduced `CheckBoxConfigurator` for improved user interactions, replacing toggle switches with checkboxes.
	- Added `MultPropertyConfigurator` for dynamic layout selection between horizontal and vertical options.
	- Enhanced `ButtonGroupConfigurator` with responsive layout adjustments and improved button styling.

- **Improvements**
	- Updated styling for various components to enhance visual organization and accessibility.
	- Introduced new CSS variables for better theming and customization of button groups and backgrounds.

- **Bug Fixes**
	- Improved logic for handling multiple property updates, ensuring better model value synchronization.

### CODE REVIEW

File: packages/common/component/ConfigItem.vue [hunk 1]
```diff
@@ -404,14 +411,27 @@ export default {
...
     const onModelUpdate = (data, shouldUpdate = true) => {
-      const preValue = bindValue.value
-      widget.value.props.modelValue = data
-      emit('update:modelValue', data)
-      if (!shouldUpdate) {
-        return
+      if (props.property.multiProperty) {
+        const array = JSON.parse(JSON.stringify(data))
+        array.forEach((item) => {
+          if (item.isChange) {
```
Comment (chilingling) [inline packages/common/component/ConfigItem.vue:417]: 这里 `ConfigItem` 不应该感知下层子组件

File: packages/common/component/ConfigItem.vue [hunk 2]
```diff
@@ -87,16 +87,18 @@
...
-            :is="VariableConfigurator"
-            v-if="isTopLayer && !onlyEdit && property.bindState !== false && !isRelatedComponents(widget.component)"
-            :model-value="widget.props.modelValue"
-            :name="widget.props.name"
-            @update:modelValue="onModelUpdate"
-          ></component>
         </div>
       </div>
     </div>
+    <div :class="['variable-bind', labelPosition === 'left' ? 'bind-left' : '']">
```
Comment (chilingling) [inline packages/common/component/ConfigItem.vue:93]:
```
<div :class="['variable-bind',  { 'bind-left':  labelPosition === 'left' }  ]">
```

**Review by chilingling — CHANGES_REQUESTED** [review]

File: packages/common/component/ConfigItem.vue (hunk 1 above)
Comment (ianxinnew) [reply packages/common/component/ConfigItem.vue:417]: 设置多属性，改变一个属性值时，另一个属性值也需同时改变，并修改scheama，在下层子组件无法处理

Comment (ianxinnew) [reply packages/common/component/ConfigItem.vue:417]: 该需求合入后待后期讨论方案评审 后在进行整改

**Pull request closed without merging**